src/@ionic-native/plugins/google-maps/index.ts

```diff
@@ -540,30 +638,18 @@ export class GoogleMaps extends IonicNativePlugin {
   pluginRef: 'plugin.google.maps.BaseArrayClass',
   repo: ''
 })
-export class BaseArrayClass<T> extends IonicNativePlugin {
-  private _objectInstance: any;
+export class BaseArrayClass<T> extends BaseClass {
 
-  constructor(initialData: T[]) {
+  constructor(initialData?: any) {
```

Comment:
Perhaps we could use `: T[] | any` to indicate that we can pass an array of the same type, or `any` (BaseArrayClass instance).